models/articles.models.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection.js");
+
+async function fetchArticleById(article_id) {
+    //const {rows} =await db.query(`SELECT article_id FROM articles`)
+    //const validIds = rows.map((row) => row.article_id)
+    console.log('model')
+    if (!Number(article_id)) return Promise.reject({ status: 400, message: 'Bad request' })
```

Comment:
I would be hesitant to handle an "bad" Id in this manner, you have already designed a psql error handler, so you shouldn't need to handle this error yourself! psql is expecting a number, if it doesn't get one it will throw it's own error, to be caught and sent to your psql handler. 

Either way is fine but I wouldn't include both! It would be good to demonstrate automated error handling in my opinion :)